nncf/common/graph/patterns/manager.py

```diff
@@ -91,6 +92,7 @@ def _filter_patterns(
         patterns_to_filter: dict[PatternNames, Callable[[], GraphPattern]],
         device: TargetDevice,
         model_type: Optional[ModelType] = None,
+        algorithm: Optional[AlgorithmType] = None,
```

Comment:
What do you think about renaming `algorithm` variable to `algorithm_type` which is analogous to `model_type`?

Reply:
Great idea, fixed